### PR TITLE
Update get-started.xml

### PR DESCRIPTION
**Pull request by Tinywan**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code style][style-guide] of this project.
- [ ] I have updated the documentation accordingly.

[style-guide]: https://wiki.apache.org/jmeter/CodeStyleGuidelines

### CODE REVIEW

File: xdocs/usermanual/get-started.xml [hunk 1]
```diff
@@ -279,7 +279,7 @@ jmeter -t test.jmx &hellip;
 <dl>
 <dt><code>DDRAW</code></dt><dd>JVM options to influence usage of direct draw, e.g. <code>-Dsun.java2d.ddscale=true</code>. Default is empty.</dd>
 <dt><code>GC_ALGO</code></dt><dd>JVM garbage collector options. Defaults to <code>-XX:+UseG1GC -XX:MaxGCPauseMillis=250 -XX:G1ReservePercent=20</code></dd>
-<dt><code>HEAP</code></dt><dd>JVM memory settings used when starting JMeter. Defaults to <code>-Xms1g -Xmx1g -XX:MaxMetaspaceSize=256m</code></dd>
+<dt><code>HEAP</code></dt><dd>JVM memory settings used when starting JMeter. Defaults to <code>-Xms1g -Xmx1g -X:MaxMetaspaceSize=256m</code></dd>
```
Comment (vlsi) [inline xdocs/usermanual/get-started.xml:282]: Could you please clarify the reason for the chnage? AFAIK, `-XX:MaxMetaspaceSize=256m` is the proper syntax

Comment (pmouawad) [reply xdocs/usermanual/get-started.xml:282]: I guess it's a typo

Comment (Tinywan) [reply xdocs/usermanual/get-started.xml:282]: `-XX:MaxMetaspaceSize=256m` and `-X:MaxMetaspaceSize=256m` that is correct?

Comment (FSchumacher) [reply xdocs/usermanual/get-started.xml:282]: As @vlsi said, the documentation currently shows the correct option. The correct one is `-XX:MaxMetaspaceSize=...`.

**Pull request closed without merging**